docker-compose.yml

```diff
@@ -146,11 +146,14 @@ services:
             AWS_SQS_QUEUE_PHPCS: ${AWS_SQS_QUEUE_PHPCS}
             AWS_SQS_REGION: ${AWS_SQS_REGION}
             AWS_SQS_VERSION: ${AWS_SQS_VERSION}
+            FIRESTORE_QUEUE_LH: ${FIRESTORE_QUEUE_LH}
```

Comment:
Why would allow this to be configurable and not hard code the collection name?

Reply:
We don't hardcode SQS.  Also, I might want to create my own Tide-based service or APP and have my collections in different places.  It does not have to be in the root of my Firestore DB.